_data/sohailg/sohailg.md

```diff
@@ -0,0 +1,8 @@
+---
+name: Sohail Gulam
+institution: Sethu Institute of Technology
+profile_pic: sohailg.jpg
```

Comment:
Hey, this should be the same name as the image in your folder. Try changing this to `IMG-20190225-WA0002.jpg` (or renaming the file itself).

Reply:
Also this image should be at least 544x544 pixels. You probably need to resize it.

Reply:
Oh, finally: can you check off the "I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed." point?

Reply:
I've updated!